bait-files/src/main/webapp/review.js

```diff
@@ -44,28 +45,40 @@ function getRequest() {
   fetch('/request' + queryString)
       .then(response => response.json())
       .then((request) => {
-          addRequestToPage(request);
+        addRequestToPage(request);
+
+        let blobKeyString = request.image;
+        if (blobKeyString != 'noKey') {
```

Comment:
Have you tested this when no image is uploaded? Just want to make sure it works.